.travis.yml

```diff
@@ -8,8 +8,10 @@ env:
   - RUBY_GC_HEAP_GROWTH_MAX_SLOTS=300000
   - RUBY_GC_HEAP_INIT_SLOTS=600000
   - RUBY_GC_HEAP_GROWTH_FACTOR=1.25
+  - RAILS_ENV=test
+  - PARALLEL_TEST_PROCESSORS=2
   matrix:
-  - TEST_SUITE=vmdb
+  - TEST_SUITE=vmdb PARALLEL='^spec/(controllers|helpers|initializers|javascripts|lib|mailers|models|presenters|requests|routing|services|task_helpers|tools|views)'
```

Comment:
Pasting this in from a Gitter convo since this thread was referenced.

I dislike scoping all ‘normal’ specs to `/vmdb` as it wouldn’t follow canonical dir structure. That means things like infer_spec_from_file_location! wouldn’t work, more custom metadata tagging by our spec helper (or verbose spec files) and (selfishly) things like editor plugins that expect a certain dir structure won’t work.

TLDR: Basically the above comment that @Fryguy said, but even more widespread.